core/chains/evm/txmgr/helpers_test.go

```diff
@@ -1,15 +1,44 @@
 package txmgr
 
-import evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+import (
+	"context"
 
-func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EthConfirmer) {
+	"github.com/smartcontractkit/chainlink/v2/common/types"
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+)
+
+func processUnstartedEthTxsNoOp[ADDR types.Hashable[ADDR]](ctx context.Context, fromAddress ADDR) (retryable bool, err error) {
+	return false, nil
+}
+
+func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EvmConfirmer) {
 	ethConfirmer.ethClient = ethClient
 }
 
-func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EthBroadcaster) {
+func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EvmBroadcaster) {
 	ethBroadcaster.resumeCallback = resumeCallback
 }
```

Comment:
```suggestion
func (ethBroadcaster *EvmBroadcaster) SetResumeCallback(resumeCallback ResumeCallback) {
	ethBroadcaster.resumeCallback = resumeCallback
}
```

Reply:
That new fn already existed on the broadcaster, so i just deleted this one